README.md

```diff
@@ -1,16 +1,37 @@
 # Dark Theme for Jenkins (Incubated project)
 
-[![Gitter](https://badges.gitter.im/jenkinsci/hackfest.svg)](https://gitter.im/jenkinsci/hackfest?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![Gitter](https://badges.gitter.im/jenkinsci/ux-sig.svg)](https://gitter.im/jenkinsci/ux-sig?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-This repository contains a new dark theme for Jenkins which is currently **under development**.
-It requires changes in the Jenkins core, and it is not ready for production use.
-Visualization is also not perfect, contributions are welcome!
+This repository contains a dark theme for Jenkins.
```

Comment:
Would it make sense to keep a disclaimer that it is under development?

Reply:
I think it's fine, its in the title of the repo, and it mostly works, at least with core components